eslint.config.js

```diff
@@ -156,10 +168,26 @@ module.exports = [
         }
     },
 
+    // JSON files
+    {
+        name: "eslint/json",
+        files: ["**/*.json"],
+        ignores: ["**/package-lock.json"],
+        language: "json/json",
+        ...json.configs.recommended
+    },
+
+    // JSONC files
+    {
+        name: "eslint/json",
+        files: [".vscode/*.json"],
```

Comment:
Do we want to lint this file? Git ignores this, So I feel ESLint should too.

https://github.com/eslint/eslint/blob/05ab812df9af044143dc25b19ba2dee39f6b80ce/.gitignore#L21

Reply:
When I ran ESLint without this, I got lint errors, but you're right, we should just ignore that directory.